CHANGELOG.md

```diff
@@ -130,6 +130,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - All exporters will be shutdown even if one reports an error (#3091)
 - Ensure valid UTF-8 when truncating over-length attribute values. (#3156)
 
+### Changed
+
+- Decode urlencoded values from the `OTEL_RESOURCE_ATTRIBUTES` environment variable. (#2963)
```

Comment:
Please move this up into the `Unreleased` section at the top.

Reply:
Updated